src/index.js

```diff
@@ -2,7 +2,7 @@ import { loadedConfigStore } from 'src/configStore'
 
 
 const loadApp = async () => {
-  const res = await fetch('config.json')
+  const res = await fetch('/config.json')
```

Comment:
Adding a slash so that if you go to a path that's more than one level deep, the app can still load and turn it into a usable one.

Reply:
Thanks! I was just looking at this issue in my review.